dev/io.openliberty.microprofile.faulttolerance.telemetry/bnd.bnd

```diff
@@ -0,0 +1,52 @@
+#*******************************************************************************
+# Copyright (c) 2018, 2024 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+openTelemetryVersion=1.35.0
+
+Bundle-Name: io.openliberty.microprofile.faulttolerance.telemetry
+Bundle-SymbolicName: io.openliberty.microprofile.faulttolerance.telemetry
+
+Bundle-Description:MicroProfile Fault Tolerance telemetery integration, version ${bVersion}
+
+WS-TraceGroup: FAULTTOLERANCE
+
+Import-Package: org.eclipse.microprofile.metrics.*;version="[1.1,3.0)", \
+        *
```

Comment:
Likely don't need this explicit range if we fix the buildpath versions as we currently only need to work with one version of everything.